sklearn/decomposition/online_lda.py

```diff
@@ -572,8 +580,29 @@ def transform(self, X):
 
         doc_topic_distr, _ = self._e_step(X, cal_sstats=False,
                                           random_init=False)
-        # normalize doc_topic_distr
+
+        return doc_topic_distr
+
+    def transform(self, X):
+        """Transform data X according to the fitted model.
+
+           .. versionchanged:: 0.18
+              *doc_topic_distr* is now normalized
+
+        Parameters
+        ----------
+        X : array-like or sparse matrix, shape=(n_samples, n_features)
+            Document word matrix.
+
+        Returns
+        -------
+        doc_topic_distr : shape=(n_samples, n_topics)
+            Document topic distribution for X.
+        """
+
```

Comment:
I feel like there's more whitespace than makes this readable, when there's an extra line between function docstring's closing `"""` on a line of its own and function code. Also blank line between calculating a value and returning it seems unhelpful for readability.